source/features/add-co-authored-by.tsx

```diff
@@ -60,6 +60,7 @@ async function init() {
 
 features.add({
 	id: 'add-co-authored-by',
+	description: 'Add co authored by',
```

Comment:
```suggestion
	description: 'Adds `Co-Authored-By` field to merge commits',
```

Reply:
It's best to reuse the text from the readme where possible. In this case:
```suggestion
	description: 'Add co-authors when merging PRs with multiple committers',
```